internal/testplanet/uplink.go

```diff
@@ -140,46 +134,46 @@ func (planet *Planet) newUplink(name string, storageNodeCount int) (*Uplink, err
 }
 
 // ID returns uplink id
-func (uplink *Uplink) ID() storj.NodeID { return uplink.Info.Id }
+func (planetUplink *Uplink) ID() storj.NodeID { return planetUplink.Info.Id }
```

Comment:
This receiver name is worse than it was before.

Reply:
we have too many uplinks. i was trying to find a way to differentiate between testplanet Uplink vs libuplink vs uplink pkg. this is what i came up with.  do you have any better suggestions?

Reply:
best I can come up with: `client`, `user`, `peer`

Reply:
I feel ok with `client` or `peer` :)

Reply:
jusst `client` or `uplinkClient`?

Reply:
just `client`